sdk/core/azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -146,16 +146,24 @@ def __len__(self):
         return self.content_length
 
     async def __anext__(self):
-        try:
-            chunk = await self.response.content.read(self.block_size)
-            if not chunk:
+        retries_remaining = True
```

Comment:
FWIW if I see `retries_remaining` I think it will be a number counting down, maybe `retry_active`?